src/main/scala/collatz/Collatz.scala

```diff
@@ -1 +1,9 @@
+def collatz(n: Int) = {
+    if (n % 2) == 0 then n/2
+    else (3 * n) + 1
+}
 
+def collatzCount(n: Int): Int = {
+    if n != 1 then 1 + collatzCount(collatz(n))
+    else 0
+}
```

Comment:
Interesting choice to leave the base case for last! I typically default to it being the first but that's just how I like it.